common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -441,6 +444,66 @@ public void run() {
         });
     }
 
+    @Override
+    public String sign(@NonNull final String input) throws ClientException {
```

Comment:
TODO determine signing strategy, or if multiple should be employed.

Reply:
Currently using `SHA256WithRSA`

Reply:
Or should this be a different IDevicePopManager?